includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -119,15 +119,18 @@ static function() use ($section) {
                 );
 
                 foreach ($section["settings"] as $setting) {
-                    $namePrefix           = $prefix ? "{$name}_" : '';
+                    $namePrefix            = $prefix ? "{$name}_" : '';
```

Comment:
huh, hij staat nu juist scheef 😩

Reply:
Ik ga ff mn ide config checken want dit is echt irritant 😢